site/src/components/pages/docs/search/search-list-item.tsx

```diff
@@ -25,16 +25,7 @@ const SearchListItem = forwardRef<HTMLButtonElement[], SearchItemProps>(
 
     return (
       <LinkButton
-        ref={(item) => {
-          if (
-            item &&
-            typeof searchListItemsRef !== "function" &&
-            searchListItemsRef?.current &&
-            !searchListItemsRef?.current?.includes(item)
-          ) {
-            searchListItemsRef.current.push(item);
-          }
-        }}
+        ref={ref}
```

Comment:
If the `SearchListItem` declares a singular ref with the corresponding type, it can be passed directly here